emukit/benchmarking/loop_benchmarking/benchmark_plot.py

```diff
@@ -74,7 +74,7 @@ def __init__(
         self.fig_handle = None
         self.x_axis = x_axis_metric_name
 
-    def make_plot(self) -> None:
+    def make_plot(self, logy: bool = False) -> None:
```

Comment:
Just one small request - can this be `log_y`? It's so much easier to read this way